scripts/logid-generator.py

```diff
@@ -92,14 +92,10 @@ def max_log_ID(max_id, prefix):
   global VERSION_SHIFT
   for logs in all_logs:
     for log in logs['logs']:
-      if max_id < int(log[2]):
+      if int(log[2]) != 0 and (int(log[2]) >= prefix and int(log[2]) < (prefix + VERSION_SHIFT)) and (max_id < int(log[2])):
```

Comment:
Please extract the log_id to a variable:

```
log_id = log.group("ID")
```

(log[2] can be accessed with the group name log.group("ID"), in the regexp we defined names for the groups)